### ISSUE TITLE
Traces: Sample JSON doesn't appear to be valid.

### ISSUE THREAD

**Issue by smcguirecn** [open-telemetry/opentelemetry.io#2855]
The sample JSON file on this page is not validating properly. 

![image](https://github.com/open-telemetry/opentelemetry.io/assets/113528444/857daa30-5694-48ec-88db-3d958fc1b305)

**Comment by smcguirecn** [comment]
For page:
https://opentelemetry.io/docs/concepts/signals/traces/

**Comment by smcguirecn** [comment]
End of file expected is to be expected. Perhaps this should be split into three separate code blocks.

**Comment by svrnm** [comment]
I don't think this is intended to be proper JSON, although one option would be to put those into a JSON array. @cartermp wdyt?

**Comment by smcguirecn** [comment]
My immediate response was to copy the whole thing and stick it into my VSCode so I could minimize parts and see how things related to each other.

This is why I think it should probably be valid JSON and you should probably have one span per code block. Just my 2c.

**Comment by cartermp** [comment]
I decided to go with splitting up the blobs and rephrasing a few things. Thanks for the report!

**Comment by smcguirecn** [comment]
Excellent! Thanks.
